File: client/src/types/media-tag.ts
export type MediaTagResponse = {
    id: string,
    name: string,
    slug: string,
    color?: string
}

export type MediaTagCreate = {
    name?: string,
    slug?: string,
    color?: string
}

export type MediaTagUpdate = {
    name?: string,
    slug?: string,
    color?: string
}

File: server (副本)/app/schemas/media_tag.py
from pydantic import BaseModel


class MediaTagCreate(BaseModel):
    name: str
    slug: str
    color: str | None = None


class MediaTagUpdate(BaseModel):
    name: str | None = None
    slug: str | None = None
    color: str | None = None


class MediaTagResponse(BaseModel):
    id: str
    name: str
    slug: str
    color: str | None = None

    model_config = {"from_attributes": True}
